### DISCUSSION TITLE
Copy drafts to the clipboard instead of discarding them.

### DISCUSSION THREAD

**Discussion by ItsIgnacioPortal** [Docile-Alligator/Infinity-For-Reddit#944] [Ideas]
A draft management system is quite a big task, so I suggest that instead of just discarding comment/post drafts, they be copied into the clipboard.

That way they can trivially be exported, restored, deleted (overwritten), etc by the user. This feature would be extremely easy to implement

**Comment by Wladefant** [comment]
better but more difficult alternative: #943

**Comment by magnus-ISU** [comment]
Please do this. I am always forgetting what the person 2 up in the comment chain said and needing to go back, hit the warning and realize I shouldn't discard, don't discard, manually copy all, go back again, confirm the warning, read the thing, and open the comment, and paste.

It would be even cooler if you could copy something like "$unsaved_comment_data ($parent_comment_url)" to the clipboard and autofill the data if the clipboard is in that format and you're replying to the same comment. But please don't let that slow down implementing this feature.

When implemented the warning should go away too and the above process becomes shorter by like 4 steps.
